Testscripts/Windows/DM-MAXMEMHONOR.ps1

```diff
@@ -183,7 +183,6 @@ function Main {
                 Throw "$vm2Name assigned memory exceeded the maximum memory set"
             }
         }
-        Stop-VM -VMName $vm2name -ComputerName $HvServer -force
```

Comment:
The whole framework will shutdown it after case pass. This is output if stop-vm in its own script, it will cost 12 mins to upload or download files from a stopped VM.
```
12/28/2019 01:25:42 : [INFO ] Test case script result does not match known ones:  PASS
12/28/2019 01:25:42 : [INFO ] ==> Upload test results to database.
12/28/2019 01:25:42 : [INFO ] ==> Check if the test target machines are still running.
12/28/2019 01:25:42 : [INFO ] Trying to connect to deployed VMs.
12/28/2019 01:25:42 : [INFO ] Connecting to 10.226.0.131 : 22 succeeded.
12/28/2019 01:26:03 : [WARN ] TCP test failed
12/28/2019 01:26:03 : [INFO ] Connecting to XX.XXX.X.XXX : 22 failed.
12/28/2019 01:26:03 : [INFO ] 1 VM(s) still waiting for port 22 open.
12/28/2019 01:26:03 : [INFO ] Retrying 1/10 in 3 seconds.
12/28/2019 01:26:06 : [INFO ] Connecting to 10.226.0.131 : 22 succeeded.
12/28/2019 01:26:27 : [WARN ] TCP test failed
12/28/2019 01:26:27 : [INFO ] Connecting to XX.XXX.X.XXX : 22 failed.
12/28/2019 01:26:27 : [INFO ] 1 VM(s) still waiting for port 22 open.
12/28/2019 01:26:27 : [INFO ] Retrying 2/10 in 3 seconds.
12/28/2019 01:26:30 : [INFO ] Connecting to 10.226.0.131 : 22 succeeded.
12/28/2019 01:26:51 : [WARN ] TCP test failed
12/28/2019 01:26:51 : [INFO ] Connecting to XX.XXX.X.XXX : 22 failed.
12/28/2019 01:26:51 : [INFO ] 1 VM(s) still waiting for port 22 open.
12/28/2019 01:26:51 : [INFO ] Retrying 3/10 in 3 seconds.
12/28/2019 01:26:54 : [INFO ] Connecting to 10.226.0.131 : 22 succeeded.
12/28/2019 01:27:15 : [WARN ] TCP test failed
12/28/2019 01:27:16 : [INFO ] Connecting to XX.XXX.X.XXX : 22 failed.
12/28/2019 01:27:16 : [INFO ] 1 VM(s) still waiting for port 22 open.
12/28/2019 01:27:16 : [INFO ] Retrying 4/10 in 3 seconds.
12/28/2019 01:27:19 : [INFO ] Connecting to 10.226.0.131 : 22 succeeded.
12/28/2019 01:27:40 : [WARN ] TCP test failed
12/28/2019 01:27:40 : [INFO ] Connecting to XX.XXX.X.XXX : 22 failed.
12/28/2019 01:27:40 : [INFO ] 1 VM(s) still waiting for port 22 open.
12/28/2019 01:27:40 : [INFO ] Retrying 5/10 in 3 seconds.
12/28/2019 01:27:43 : [INFO ] Connecting to 10.226.0.131 : 22 succeeded.
12/28/2019 01:28:04 : [WARN ] TCP test failed
12/28/2019 01:28:04 : [INFO ] Connecting to XX.XXX.X.XXX : 22 failed.
12/28/2019 01:28:04 : [INFO ] 1 VM(s) still waiting for port 22 open.
12/28/2019 01:28:04 : [INFO ] Retrying 6/10 in 3 seconds.
12/28/2019 01:28:07 : [INFO ] Connecting to 10.226.0.131 : 22 succeeded.
12/28/2019 01:28:28 : [WARN ] TCP test failed
12/28/2019 01:28:28 : [INFO ] Connecting to XX.XXX.X.XXX : 22 failed.
12/28/2019 01:28:28 : [INFO ] 1 VM(s) still waiting for port 22 open.
12/28/2019 01:28:28 : [INFO ] Retrying 7/10 in 3 seconds.
12/28/2019 01:28:31 : [INFO ] Connecting to 10.226.0.131 : 22 succeeded.
12/28/2019 01:28:52 : [WARN ] TCP test failed
12/28/2019 01:28:52 : [INFO ] Connecting to XX.XXX.X.XXX : 22 failed.
12/28/2019 01:28:52 : [INFO ] 1 VM(s) still waiting for port 22 open.
12/28/2019 01:28:52 : [INFO ] Retrying 8/10 in 3 seconds.
12/28/2019 01:28:55 : [INFO ] Connecting to 10.226.0.131 : 22 succeeded.
12/28/2019 01:29:16 : [WARN ] TCP test failed
12/28/2019 01:29:16 : [INFO ] Connecting to XX.XXX.X.XXX : 22 failed.
12/28/2019 01:29:16 : [INFO ] 1 VM(s) still waiting for port 22 open.
12/28/2019 01:29:16 : [INFO ] Retrying 9/10 in 3 seconds.
12/28/2019 01:29:19 : [INFO ] Connecting to 10.226.0.131 : 22 succeeded.
12/28/2019 01:29:40 : [WARN ] TCP test failed
12/28/2019 01:29:40 : [INFO ] Connecting to XX.XXX.X.XXX : 22 failed.
12/28/2019 01:29:40 : [INFO ] 1 VM(s) still waiting for port 22 open.
12/28/2019 01:29:40 : [INFO ] Retrying 10/10 in 3 seconds.
12/28/2019 01:29:43 : [INFO ] ==> Run test cleanup script if defined.
12/28/2019 01:29:43 : [INFO ] Start to do cleanup for case DYNAMIC-MEMORY-MAX-MEMORY-HONORED
12/28/2019 01:29:43 : [INFO ] Removing Background Job 10563...
12/28/2019 01:29:43 : [INFO ] Removing all files logs from IP : 10.226.0.131 PORT : 22
12/28/2019 01:29:43 : [DEBUG] dos2unix: converting file C:\XXXXv2\EX56\TestResults\2019-26-12-22-49-51-7170\DYNAMIC-MEMORY-MAX-MEMORY-HONORED\runtest.sh to Unix format...
12/28/2019 01:29:43 : [DEBUG] Uploading C:\XXXXv2\EX56\TestResults\2019-26-12-22-49-51-7170\DYNAMIC-MEMORY-MAX-MEMORY-HONORED\runtest.sh to XXXX @ 10.226.0.131 : 22 using password authentication
12/28/2019 01:29:44 : [DEBUG] Upload successful after 1 attempt
12/28/2019 01:29:44 : [DEBUG] .\Tools\plink.exe -ssh -t -pw ****** -P 22 XXXX@10.226.0.131 "echo ****** | sudo -S rm -rf *"
12/28/2019 01:29:45 : [DEBUG] rm -rf * executed successfully in 0.77 seconds.
12/28/2019 01:29:45 : [INFO ] All files removed from current user XXXX home folder successfully. VM IP : 10.226.0.131 PORT : 22
12/28/2019 01:29:45 : [INFO ] Removing all files logs from IP : XX.XXX.X.XXX PORT : 22
12/28/2019 01:29:45 : [DEBUG] dos2unix: converting file C:\XXXXv2\EX56\TestResults\2019-26-12-22-49-51-7170\DYNAMIC-MEMORY-MAX-MEMORY-HONORED\runtest.sh to Unix format...
12/28/2019 01:29:45 : [DEBUG] Uploading C:\XXXXv2\EX56\TestResults\2019-26-12-22-49-51-7170\DYNAMIC-MEMORY-MAX-MEMORY-HONORED\runtest.sh to XXXX @ XX.XXX.X.XXX : 22 using password authentication
12/28/2019 01:30:28 : [WARN ] Error in upload, attempt 1, retrying
12/28/2019 01:30:38 : [DEBUG] Uploading C:\XXXXv2\EX56\TestResults\2019-26-12-22-49-51-7170\DYNAMIC-MEMORY-MAX-MEMORY-HONORED\runtest.sh to XXXX @ XX.XXX.X.XXX : 22 using password authentication
12/28/2019 01:31:21 : [WARN ] Error in upload, attempt 2, retrying
12/28/2019 01:31:32 : [DEBUG] Uploading C:\XXXXv2\EX56\TestResults\2019-26-12-22-49-51-7170\DYNAMIC-MEMORY-MAX-MEMORY-HONORED\runtest.sh to XXXX @ XX.XXX.X.XXX : 22 using password authentication
12/28/2019 01:32:15 : [WARN ] Error in upload, attempt 3, retrying
12/28/2019 01:32:25 : [DEBUG] Uploading C:\XXXXv2\EX56\TestResults\2019-26-12-22-49-51-7170\DYNAMIC-MEMORY-MAX-MEMORY-HONORED\runtest.sh to XXXX @ XX.XXX.X.XXX : 22 using password authentication
12/28/2019 01:33:08 : [WARN ] Error in upload, attempt 4, retrying
12/28/2019 01:33:18 : [DEBUG] Uploading C:\XXXXv2\EX56\TestResults\2019-26-12-22-49-51-7170\DYNAMIC-MEMORY-MAX-MEMORY-HONORED\runtest.sh to XXXX @ XX.XXX.X.XXX : 22 using password authentication
12/28/2019 01:34:02 : [WARN ] Error in upload, attempt 5, retrying
12/28/2019 01:34:12 : [DEBUG] Uploading C:\XXXXv2\EX56\TestResults\2019-26-12-22-49-51-7170\DYNAMIC-MEMORY-MAX-MEMORY-HONORED\runtest.sh to XXXX @ XX.XXX.X.XXX : 22 using password authentication
12/28/2019 01:34:55 : [WARN ] Error in upload, attempt 6, retrying
12/28/2019 01:35:05 : [DEBUG] Uploading C:\XXXXv2\EX56\TestResults\2019-26-12-22-49-51-7170\DYNAMIC-MEMORY-MAX-MEMORY-HONORED\runtest.sh to XXXX @ XX.XXX.X.XXX : 22 using password authentication
12/28/2019 01:35:48 : [WARN ] Error in upload, attempt 7, retrying
12/28/2019 01:35:59 : [DEBUG] Uploading C:\XXXXv2\EX56\TestResults\2019-26-12-22-49-51-7170\DYNAMIC-MEMORY-MAX-MEMORY-HONORED\runtest.sh to XXXX @ XX.XXX.X.XXX : 22 using password authentication
12/28/2019 01:36:42 : [WARN ] Error in upload, attempt 8, retrying
12/28/2019 01:36:52 : [DEBUG] Uploading C:\XXXXv2\EX56\TestResults\2019-26-12-22-49-51-7170\DYNAMIC-MEMORY-MAX-MEMORY-HONORED\runtest.sh to XXXX @ XX.XXX.X.XXX : 22 using password authentication
12/28/2019 01:37:35 : [WARN ] Error in upload, attempt 9, retrying
12/28/2019 01:37:46 : [DEBUG] Uploading C:\XXXXv2\EX56\TestResults\2019-26-12-22-49-51-7170\DYNAMIC-MEMORY-MAX-MEMORY-HONORED\runtest.sh to XXXX @ XX.XXX.X.XXX : 22 using password authentication
```